src/fidesops/analytics.py

```diff
@@ -0,0 +1,70 @@
+import logging
+import os
+from platform import system
+from typing import Dict
+
+from fideslog.sdk.python.client import AnalyticsClient
+from fideslog.sdk.python.event import AnalyticsEvent
+from fideslog.sdk.python.exceptions import AnalyticsError
+from fideslog.sdk.python.utils import FIDESOPS, generate_client_id
+
+from fidesops.core.config import config, update_config_file
+
+from ._version import get_versions
+
+logger = logging.getLogger(__name__)
+
+
+def get_version() -> str:
+    """Version of Fidesops"""
+    return get_versions()["version"]
```

Comment:
Rather than reimplement this, you can do:
```python
from fidesops import __version__ as fidesops_version
```

Reply:
ahh good point